spring-cloud-dataflow-server-cloudfoundry-docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -69,7 +69,7 @@ deployment to fail
 
 [subs=attributes]
 ```
-cf push dataflow-server -m 2G -k 2G --no-start -p spring-cloud-dataflow-server-cloudfoundry-{project-version}.jar
+cf push dataflow-server -b java_buildpack_offline -m 2G -k 2G --no-start -p spring-cloud-dataflow-server-cloudfoundry-{project-version}.jar
```

Comment:
I believe `-b java_buildpack` is the default value in a PCF install.  Let's align with that default value for the docs.